endpoints.go

```diff
@@ -2,18 +2,27 @@ package main
 
 import (
 	"encoding/json"
+	"io/ioutil"
 	"fmt"
 	"net/http"
 	"time"
+	"strings"
 
+    "github.com/ONSdigital/ras-rm-survey/logger"
+    "github.com/gofrs/uuid"
 	"github.com/ONSdigital/ras-rm-survey/models"
 	"github.com/gorilla/mux"
 	"github.com/spf13/viper"
 )
 
 func handleEndpoints(r *mux.Router) {
```

Comment:
The non-info endpoints should have basic auth on them. I don't mind missing it out for now so I've added a card to refinement to add it